--- ftgl/src/FTVectoriser.cpp	2026-08-10 08:59:56.547220008 -0000
+++ ftgl/src/FTVectoriser.cpp	2026-08-10 09:02:11.581010988 -0000
@@ -168,7 +168,7 @@
     for(int i = 0; i < ftContourCount; ++i)
     {
         FT_Vector* pointList = &outline.points[startIndex];
-        char* tagList = &outline.tags[startIndex];
+        char* tagList = (char *)&outline.tags[startIndex];

         endIndex = outline.contours[i];
         contourLength =  (endIndex - startIndex) + 1;
